data-manipulation/encryption/chaskey/encrypt-data-using-chaskey.yml

```diff
@@ -0,0 +1,42 @@
+rule:
+  meta:
+    name: encrypt data using chaskey
+    namespace: data-manipulation/encryption/chaskey
+    authors:
+      - still@teamt5.org
+    scopes:
+      static: function
```

Comment:
Can this (or a subset of features) be tightened with _basic block_ to avoid potential FPs?

Reply:
Ditto with speck comment.